### ISSUE TITLE
Maintenance of the module

### ISSUE THREAD

**Issue by ddufresne** [akretion/overdue-reminder#4]
Hello @alexis-via 

Thank you for the overdue-reminder module, it is very appreciated from our clients.

We have an issue with running reminders in batch.

I am wondering if you are actively maintaining/using the module.
I would like to propose code fixes and tests for the module.

Would you review the PR?

**Comment by alexis-via** [comment]
Yes, I'm still maintaining actively the reminder module, at least on v10, v12 and v14 (I usually work only on even versions). I'm glad that you like it.
I'll review your PR for improvements/fixes/tests. If possible, please contribute on v14 (fixes can be backported later to v12 and v10)

**Comment by ddufresne** [comment]
@alexis-via ok, very well
